src/main/java/net/mtrop/doom/tools/WTexScanMain.java

```diff
@@ -84,6 +86,7 @@
 		private boolean outputFlats;
 		private boolean skipSkies;
 		private List<File> wadFiles;
+		private List<String> mapsToScan;
```

Comment:
I think this should be a `Set`, as that would set (heh) a better foundation for improving the search running time if multiple maps are specified.